tests/test_inventory_items.py

```diff
@@ -97,20 +97,6 @@ def test_serialize(self):
         self.assertIn("in_stock", data)
         self.assertEqual(data["in_stock"], item.in_stock)
 
-    def test_create_a_inventory_item(self):
```

Comment:
I just moved this test next to the test for `delete` behavior so it's easier to read.